tests/ui/raw-ref-op/never-place-isnt-diverging.rs

```diff
@@ -0,0 +1,13 @@
+#![feature(never_type)]
+
+fn make_up_a_value<T>() -> T {
+    unsafe {
```

Comment:
After #129248, we can take away the `unsafe {}` so it's especially important this turn into an error :)